code/modules/vending/wardrobes.dm

```diff
@@ -35,11 +35,19 @@
 					/obj/item/clothing/under/rank/security/officer/blueshirt = 3,
 					/obj/item/clothing/under/rank/security/officer/mallcop = 3,
 					/obj/item/clothing/neck/tie/red = 6,
-					/obj/item/clothing/neck/tie/black = 6,)
+					/obj/item/clothing/neck/tie/black = 6,
+					/obj/item/clothing/suit/armor/secduster = 3,
+					/obj/item/clothing/head/helmet/hat/cowboy = 3)
 	contraband = list(/obj/item/clothing/suit/hooded/wintercoat/security/old = 3)
 	premium = list(/obj/item/clothing/under/rank/security/officer/formal = 3,
 					/obj/item/clothing/suit/security/officer = 3,
-					/obj/item/clothing/head/beret/sec/navyofficer = 3)
+					/obj/item/clothing/head/beret/sec/navyofficer = 3,
+					/obj/item/clothing/suit/armor/secduster/engineering = 2,
+					/obj/item/clothing/head/helmet/hat/cowboy/engineering = 2,
+					/obj/item/clothing/suit/armor/secduster/medical = 2,
+					/obj/item/clothing/head/helmet/hat/cowboy/medical = 2,
+					/obj/item/clothing/suit/armor/secduster/science = 2,
+					/obj/item/clothing/head/helmet/hat/cowboy/science = 2)
```

Comment:
looks like you are missing the cargo ones

Reply:
Fixed!  Thanks, I have no idea how I missed that lol